### PR TITLE
EX-SU-Pset_ConcreteElementGeneralUpdate

### PR DESCRIPTION
**Pull request by larswik**
Two extra properties
Added IfcArchElement applicability

### CODE REVIEW

**Comment by larswik** [comment]
@SergejMuhic : Maybe a quick look here since it is a change of an existing Pset?

**Review by SergejMuhic — COMMENTED** [review]
One remark, despite me knowing how hard it is to get technical documentation at this stage, should we add documentation to the three new concrete casting methods?

File: IFC4x3/PropertyConstants/s/SLIPFORMED_1EMIC6hL56iBaWAE3$42Ol.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SLIPFORMED_1EMIC6hL56iBaWAE3$42Ol" Name="SLIPFORMED" UniqueId="4e592306-ad51-46b0-b920-28e0ff10262f" />
```
Comment (SergejMuhic) [inline IFC4x3/PropertyConstants/s/SLIPFORMED_1EMIC6hL56iBaWAE3$42Ol.xml]: Wiki: "Slip forming, continuous poured, continuously formed, or slipform construction is a construction method in which concrete is poured into a continuously moving form."
or
https://theconstructor.org/construction/slipform-construction-uses/185/: "The slipform helps to conduct continuous pouring of the concrete to the moving formwork."
```suggestion
<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SLIPFORMED_1EMIC6hL56iBaWAE3$42Ol" Name="SLIPFORMED" UniqueId="4e592306-ad51-46b0-b920-28e0ff10262f" />
```

File: IFC4x3/PropertyConstants/s/SPRAYED_DRY_3zQuFIQc5D_AZg9HF7zun_.xml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SPRAYED_DRY_3zQuFIQc5D_AZg9HF7zun_" Name="SPRAYED_DRY" UniqueId="fd6b83d2-6a61-4df8-a8ea-2513c7f78c7e" />
```
Comment (SergejMuhic) [inline IFC4x3/PropertyConstants/s/SPRAYED_DRY_3zQuFIQc5D_AZg9HF7zun_.xml]: https://hydrokarst.fr/en/sprayed-concrete/: "a blend of cement and sand, without additional water. The water is added directly at the end of the lance to create the concrete spray. This dry-mix method meets certain spray distance restrictions. Dry-mix offers greater spraying distances than wet-mix. In certain cases, this method also helps manage the concrete preparation and deployment times."

File: IFC4x3/PropertyConstants/s/SPRAYED_WET_1K7hY64hDBQe59hbmdY$B6.xml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SPRAYED_WET_1K7hY64hDBQe59hbmdY$B6" Name="SPRAYED_WET" UniqueId="541eb886-12b3-4b6a-8149-ae5c278bf2c6" />
```
Comment (SergejMuhic) [inline IFC4x3/PropertyConstants/s/SPRAYED_WET_1K7hY64hDBQe59hbmdY$B6.xml]: same source as above: "the concrete mix is watered and machine-mixed then pumped through a pipe to the application nozzle."

**Review by SergejMuhic — COMMENTED** [review]
Proposals for descriptions.

**Comment by larswik** [comment]
@SergejMuhic : I added the suggested descriptions. Check and merge.

File: IFC4x3/PropertyConstants/s/SLIPFORMED_1EMIC6hL56iBaWAE3$42Ol.xml (hunk 1 above)
Comment (larswik) [reply IFC4x3/PropertyConstants/s/SLIPFORMED_1EMIC6hL56iBaWAE3$42Ol.xml]: Added description in [03a28cc](https://github.com/bSI-InfraRoom/IFC-Specification/pull/824/commits/03a28cc8c6229008d4d538438d999ed34110178f)

File: IFC4x3/PropertyConstants/s/SPRAYED_DRY_3zQuFIQc5D_AZg9HF7zun_.xml (hunk 2 above)
Comment (larswik) [reply IFC4x3/PropertyConstants/s/SPRAYED_DRY_3zQuFIQc5D_AZg9HF7zun_.xml]: Added description in [03a28cc](https://github.com/bSI-InfraRoom/IFC-Specification/pull/824/commits/03a28cc8c6229008d4d538438d999ed34110178f)

File: IFC4x3/PropertyConstants/s/SPRAYED_WET_1K7hY64hDBQe59hbmdY$B6.xml (hunk 3 above)
Comment (larswik) [reply IFC4x3/PropertyConstants/s/SPRAYED_WET_1K7hY64hDBQe59hbmdY$B6.xml]: Added description in [03a28cc](https://github.com/bSI-InfraRoom/IFC-Specification/pull/824/commits/03a28cc8c6229008d4d538438d999ed34110178f)

**Review by SergejMuhic — APPROVED** [review]

**Pull request merged**